puzzle/server/blueprints/variants/templates/sv_variants.html

```diff
@@ -42,10 +43,21 @@ <h4 class="panel-title">
             <td>{{ variant.cytoband_start or '-' }}</td>
             <td>{{ variant.cytoband_stop or '-' }}</td>
             <td align="right">{{ '{0:,}'.format(variant.sv_len) }}</td>
+	    <td> 
```

Comment:
Exactly! :-) This is often implicit in score/rank, but helps in sifting through variants w/o clicking them.